editor/editor_themes.cpp

```diff
@@ -1393,7 +1392,6 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	style_tooltip->set_bg_color(dark_color_3 * Color(0.8, 0.8, 0.8, 0.9));
 	style_tooltip->set_border_width_all(0);
 	theme->set_color("font_color", "TooltipLabel", font_hover_color);
-	theme->set_color("font_color_shadow", "TooltipLabel", Color(0, 0, 0, 0));
```

Comment:
It's not unused, but it was missed in some rename. Should be `font_shadow_color`. Default theme has this defined correctly.